tox.ini

```diff
@@ -7,14 +7,22 @@
 envlist = py27, py35, py36, py37, py38, {py35, py36, py37, py38}-async
 
 [testenv]
+extras = async: async
+
 commands = coverage erase
            !async: coverage run -m unittest discover -p "test_*.py" []
-           async:  coverage run -m unittest discover -p "*test_*.py" []
-           !py27:  coverage report
-           py27:   coverage report --ignore-errors
+           async: coverage run -m unittest discover -p "*test_*.py" []
+           coverage report --omit="sendgrid/async_sendgrid.py"i
```

Comment:
It must be omit in non-async tests only. The fix will make `commands` section in `py27` env obsolete.